src/javascript/app/pages/user/warning_scam_message.js

```diff
@@ -0,0 +1,50 @@
+const getElementById       = require('../../../_common/common_functions').getElementById;
+const BinarySocket         = require('../../base/socket');
+const Client               = require('../../base/client');
+const State                = require('../../../_common/storage').State;
+
+const WarningScamMessage = (()=>{
+    const urlParams = new URLSearchParams(window.location.search);
+    const actionParams = urlParams.get('action');
+    let warning_scam_message, warning_scam_message_button, warning_scam_message_checkbox;
+
+    if (actionParams === 'signup') {
+        localStorage.setItem('new_user_account', true);
+    }
+
+    const has_read_warning_scam_message = () => {
+        BinarySocket.wait('authorize', 'website_status').then(()=> {
+            const is_logged_in = !!Client.isLoggedIn();
+            const is_brazil = State.getResponse('website_status.clients_country') === 'br';
+            const is_message_read = localStorage.getItem('read_scam_message') || false;
+            const is_new_account = localStorage.getItem('new_user_account') || false;
```

Comment:
```suggestion
            const is_message_read = !!localStorage.getItem('read_scam_message');
            const is_new_account = !!localStorage.getItem('new_user_account');
```
Doing `!!` will coerce null values to false